### ISSUE TITLE
django-smart-selects not working in jet

### ISSUE THREAD

**Issue by neotrons** [geex-arts/django-jet#67]
How to create a cascade select in django jet? I am integrate django-smart-selects in my admin but not working. when i use django-admin default not has problem.

**Comment by SalahAdDin** [comment]
Are you shure that isn't problem with django-smart-selects? I say because i'm using it and i don't know if the problem is that django-jet doesn't user normal input selects or if a problem with the current django-smart-selects, i report an issue there.

**Comment by neotrons** [comment]
Hi sir thanks for your time what version do you use?

I use this version

```
Django==1.8.3
django-jet==0.1.4
django-smart-selects==1.2.2
```

thanks

**Comment by SalahAdDin** [comment]
I have these version but i'm working with custom branch, but, if the lastest version doesn't works i think that is a smart-selects problem.

**Comment by neotrons** [comment]
thanks sir, please 1 questions. how i can access the change event (jquery) in the select.

i use but not fount the event

$("#id_selectname").change(function(e){
    alert("test");
});

thanks.

**Comment by vitorfragoso** [comment]
I also think there is a problem between Jet and Smart Select. I was using Smart Select successfully before install the Jet template. After install Jet, the Smart Select has stopped working.

In my case, I have two fields that are: team_name and employee_name. The employee_name should be loaded based on team_name.

Something like that:
	team_name = models.ForeignKey(Team)
	employee_name = ChainedForeignKey(
		Employee,
		chained_field="team_name",
		chained_model_field="team_name", 
		show_all=False, 
		auto_choose=True,
		unique= True)

Now the problem is that the employee_name field is not loaded after select the team_name.

**Comment by SalahAdDin** [comment]
I have the same problem.

**Comment by vitorfragoso** [comment]
I have observed the following behavior:
 - Field B depends on Field A.
 - Both fields are required.
When I fill out **Field A**, keep the **Field B** as blank and try to save Then **Field B** is loaded.
After this behavior, I can fill out **Field B** and finally save.

I believe we should call a specific JS when a user click on Add/Edit New Item. It seems to be a JS that we are calling after try to save.

**Comment by hakimkal** [comment]
I have realised the reason is django-jet (0.1.2) uses jquery 1.11.3 and django-smart-selects uses jquery 2.2.0. this is the conflict point, hence from django admin area, it creates conflict.

**Comment by SalahAdDin** [comment]
@hakimkal, are you sure about it? Have you a trace back of this?

**Comment by hakimkal** [comment]
@SalahAdDin  yes, I am very sure. i had to revert to the default django-admin UI to use django smart-select.

**Comment by SalahAdDin** [comment]
Will be very useful that you put here a tracebak.

**Comment by SalahAdDin** [comment]
@hakimkal so bad, do you think if problem is with oudated `jquery` version? or do you think is for newest `jquery` version?

**Comment by scrummastermind** [comment]
I have raised a PR, in django-smart-selects to fix this issue, feel free to use forked repo
https://github.com/digi604/django-smart-selects/pull/209

**Comment by scrummastermind** [comment]
This is  JQuey binding issue, jet uses django jQuery while smart select uses google api  by default

**Comment by Ismael-VC** [comment]
Please come to the django-jet Discord server so we can organize if you like:

  *   https://discord.gg/3yr5yeH

Welcome! 😄
